src/libtools/sdl2rwops.c

```diff
@@ -85,36 +85,36 @@ EXPORT int32_t my2_native_close(SDL2_RWops_t *context)
 }
 EXPORT int64_t my2_emulated_size(SDL2_RWops_t *context)
 {
-    return (int64_t)RunFunction(my_context, (uintptr_t)context->hidden.my.orig->size, 1, context->hidden.my.orig);
+    return (int64_t)RunFunctionFmt(my_context, (uintptr_t)context->hidden.my.orig->size, "p", context->hidden.my.orig);
 }
 EXPORT int64_t my2_emulated_seek(SDL2_RWops_t *context, int64_t offset, int32_t whence)
 {
-    return (int64_t)RunFunction(my_context, (uintptr_t)context->hidden.my.orig->seek, 3, context->hidden.my.orig, offset, whence);
+    return (int64_t)RunFunctionFmt(my_context, (uintptr_t)context->hidden.my.orig->seek, "pLi", context->hidden.my.orig, offset, whence);
```

Comment:
should be "pli" because it's signed long

Reply:
also, probably more like "pIi" because it's an explicit 64bits value iirc

Reply:
There must be a lot of errors like this, you might want to review after I double-checked.

Reply:
I'm afk for an hour or so, I'll do the check later.